### ISSUE TITLE
Pre-training and applying for node classification

### ISSUE THREAD

**Issue by timilsinamohan** [GentleZhu/EGI#1]
Hi thanks for this great library :-) !

I have two graphs , I want to train large graph (A) and then do the transfer learning in target graph (B) for node classification task. It is something like you have shown in the paper: https://arxiv.org/pdf/2009.05204.pdf, at Table 7,  for training in the European airport (source) and transferring it to USA(target) and Brazil airport(target). 

I run the code like this to run EGI model:
`python run_airport.py --data-id="data/europe-airports.edgelist" --data-src="data/europe-airports.edgelist" --file-path=data/usa-airports.edgelist --label-path=data/labels-usa-airports.txt --n-dgi-epochs=100  --n-hidden=32 --self-loop --gpu=0 --n-layers=1 --dgi-lr=0.01 --model-id=2 --model-type=2`
It reports me the accuracy: `Test Accuracy 0.6319, std 0.0161` 

I presume **--data-src** argument is for the source graph parameter. Is this correct? Don't we need the labels for the source graph?  Did I miss something? If it is not could you kindly guide me which code is to use for the pre-training and then apply transfer learning ?

Thanks in advance,
Mohan

**Comment by Skyorca** [comment]
--label-path=data/labels-usa-airports.txt is the path for source labels which you have already loaded
